tests/test/diff-id.js

```diff
@@ -216,5 +216,288 @@ describe('diff-id', function () {
         })
       expect(r2.status).to.equal(400)
     })
+
+    it('diff everything between two commits', async function () {
+      const class1 = util.randomString()
+      const class2 = util.randomString()
+      await document
+        .insert(agent, docPath, {
+          schema: [{ '@type': 'Class', '@id': class1, a: 'xsd:string', b: class2 },
+            {
+              '@type': 'Class',
+              '@id': class2,
+              c: 'xsd:integer',
+              '@subdocument': [],
+              '@key': { '@type': 'Lexical', '@fields': ['c'] },
+            },
+          ],
+        })
+        .then(document.verifyInsertSuccess)
+      const r1 = await document
+        .insert(agent, docPath, {
+          instance: {
+            '@type': class1,
+            a: 'pickles and eggs',
+            b: {
+              '@type': class2,
+              c: 3,
+            },
+          },
+        })
+        .then(document.verifyInsertSuccess)
+      const dv1 = r1.header['terminusdb-data-version']
+      const [docId] = r1.body
+      const r2 = await document
+        .replace(agent, docPath, {
+          instance: {
+            '@type': class1,
+            a: 'pickles and eggs',
+            '@id': docId,
+            b: {
+              '@type': class2,
+              c: 4,
+            },
+          },
+        })
+        .then(document.verifyInsertSuccess)
+      const dv2 = r2.header['terminusdb-data-version']
+
+      const { path } = endpoint.versionDiff(agent.defaults())
+
+      const r3 = await agent.post(path).send(
+        {
+          before_data_version: dv1,
+          after_data_version: dv2,
+        })
+      expect(r3.status).to.equal(200)
+    })
+
+    it('diff unchanged', async function () {
```

Comment:
This name is not what the test does. It checks that an insert is displayed in the diff as such.